bb.edn

```diff
@@ -0,0 +1,110 @@
+{:min-bb-version "0.8.0"
+ :pods {;since babashka 0.8.0, can only be declared in bb.edn
+        org.babashka/tools-deps-native {:version "0.0.5"}
+        clj-kondo/clj-kondo {:version "2022.10.05"}}
+ :deps {datahike/bb {:local/root "bb"}}
+ :tasks {:requires [[clojure.edn :as edn]
+                    [pod.borkdude.clj-kondo :as clj-kondo]
+                    [tools.build :as build]
+                    [tools.cljdoc :as cljdoc]
+                    [tools.deps :as deps]
+                    [tools.deploy :as deploy]
+                    [tools.release :as relase]
+                    [tools.test :as test]
+                    [tools.version :as version]]
+         :init (do (def config-file "config.edn")
+                   (def config (edn/read-string (slurp config-file)))
+                   (def deps (edn/read-string (slurp "deps.edn"))))
+         ;; hook :enter could be used to load config according to project
+         
+         ; tools
+         
+         bench {:doc "Run benchmarks"
+                :task (clojure "-M:benchmark" "measure")} 
+         
+         docs {:doc "Get a preview on the clojure docs"
+               :depends [jar install]
+               :task (cljdoc/preview config)}
+         
+         ffix {:doc "Format source files" 
+               :task (clojure "-M:ffix")}
+         
+         rpl {:doc "Start a repl"
+              :task (clojure "-M:repl")}
+
+
+         ; checks
+         
+         format {:doc "Test formatting"
+                 :task (clojure "-M:format")}
+
+         lint {:doc "Run clj-kondo linter"
+               :require [[pod.borkdude.clj-kondo :as clj-kondo]]
+               :task (clj-kondo/print! (clj-kondo/run! {:lint "."}))}
+
+         test {:doc "Run all tests or restrict to 'native-image', 'back-compat' or a kaocha test id (see tests.edn)"
+               :depends [compile]
+               :task (apply test/-main config *command-line-args*)}
+
+         kaocha {:doc "Run kaocha with arbitrary arguments"
+                 :task (apply test/kaocha *command-line-args*)}
+         
+         deps {:doc "Check consistency of dependencies"
+               :task (if *command-line-args*
+                       (apply deps/-main *command-line-args*)
+                       (deps/check {:alias (-> deps :aliases (dissoc :datomic :1.9 :1.10) keys)}))}
```

Comment:
I only copied the tool from the tools folder. I don't really get why it exists. It seems only to check if all the aliases use the same version of a library. 

IMO this could be dropped if we checked that we always use the newest version, i.e. the task "outdated".

Reply:
I discussed it with Konrad and we are dropping this tool. We could still retrieve it from the history if we want it back.